src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/authentication")
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public void register(@RequestBody UserRequestDto userRequestDto) {

```

Comment:
Let's use an annotation @Valid here.  Spring should move this object to validator, before doing something

Reply:
Agree